5.0/en/0x12-V3-Session-management.md

```diff
@@ -93,13 +93,17 @@ Attackers start a half-open attack by attempting to lock, reset, or recover a cr
 
 ## V3.8 Session Termination
 
+Session termination may be handled either by the application itself or by the SSO provider if the SSO provider is handling session management instead of the application.
+
+Session termination should result in requiring re-authentication and be effective across the application, federated login (if present), and any relying parties.
+
 | # | Description | L1 | L2 | L3 | CWE | [NIST &sect;](https://pages.nist.gov/800-63-3/sp800-63b.html) |
 | :---: | :--- | :---: | :---: | :---: | :---: | :---: |
-| **3.8.1** | [MODIFIED, MOVED FROM 3.3.1] Verify that logout and expiration invalidate the session token, such that the back button or a downstream relying party cannot resume an authenticated session. ([C6](https://owasp.org/www-project-proactive-controls/#div-numbering)) | ✓ | ✓ | ✓ | 613 | 7.1 |
-| **3.8.2** | [MODIFIED, LEVEL L2 > L1, MOVED FROM 3.3.3] Verify that the application gives the option to terminate all other active sessions after a successful change or removal of any authentication factor (including password change via reset or recovery and, if present, an MFA settings update). This must be effective across the application, federated login (if present), and any relying parties. | ✓ | ✓ | ✓ | 613 | |
-| **3.8.3** | [MOVED FROM 3.3.4] Verify that users are able to view and (having re-entered login credentials) log out of any or all currently active sessions and devices. | | ✓ | ✓ | 613 | 7.1 |
+| **3.8.1** | [MODIFIED, MOVED FROM 3.3.1] Verify that logout and expiration terminate the user's session, such that the back button or a downstream relying party cannot resume an authenticated session. ([C6](https://owasp.org/www-project-proactive-controls/#div-numbering)) | ✓ | ✓ | ✓ | 613 | 7.1 |
+| **3.8.2** | [MODIFIED, LEVEL L2 > L1, MOVED FROM 3.3.3] Verify that the application gives the option to terminate all other active sessions after a successful change or removal of any authentication factor (including password change via reset or recovery and, if present, an MFA settings update). | ✓ | ✓ | ✓ | 613 | |
+| **3.8.3** | [MOVED FROM 3.3.4] Verify that users are able to view and (having re-entered login credentials) terminate any or all currently active sessions and devices. | | ✓ | ✓ | 613 | 7.1 |
```

Comment:
add MODIFIED tag
this is the only requirement to mention devices - doing so, it is hidden requirement for tracking devices. I prefer to remove "and devices".

Reply:
In a traditional session we "terminate" that session. In a JWT stateless mechanism, we "revoke" a session. Can we please change "terminate" to "terminate or revoke" a session to account for both architecture types?

Reply:
my point of view: "terminate session" is logic and functionality to provide. "revoke" is a technical term to make JWT invalid, and from a functionality point of view, it means terminating a session, although it is not called a session (it is still a server side state handling).

Reply:
Addressed these three comments including adding a comment at the start of the section.